src/Tests/SNIPPETS.cs

```diff
@@ -365,5 +361,49 @@ operation DependsOnChemistryWorkspace() : ((JordanWignerEncodingData, Int, Doubl
     }
 ";
 
+        public static string ValidParameterTypes =
+@"
+    operation ValidParameterTypes(
```

Comment:
Please make sure that these snippets follow Q# style guide as best as reasonable. In particular, please use verbs for operation names. (The above case should do so as well, to be fair.)

Reply:
done